joomla/joomla-cms
# Attempt to prevent information disclosure

For example if someone is looking over your shoulder while you have backup oteps shown. This little modification tries to mitigate this possibility.
One side note. The details text should be a language string.

Pull Request for Issue #37969.

### Summary of Changes

add details/summary html5 tags

### Testing Instructions

Enable 2FA in administrator for a user in Registered group
Login in frontend with this Registered user
Go to edit profile
Scroll down to see Oteps backup codes

### Actual result BEFORE applying this Pull Request

Oteps backup codes show immediately and might disclose information

### Expected result AFTER applying this Pull Request

Oteps Backup codes don't show immediately. Rather, they show only if we click on Details

### Documentation Changes Required
Possibly.
